### PR TITLE
Feature/inba 863 ugly calendar

### PR DESCRIPTION
**Pull request by morleegan**
If this PR fixes a bug, you _must_ add test cases representative of the bug.

Please refer to [Tettra](https://app.tettra.co/teams/amida/pages/amida-pull-request-and-code-review-guide) for PR review guidelines.

#### What does this PR do?
Changes the calendar to the Airbnb style calendar. 

#### Related JIRA tickets:
https://jira.amida.com/browse/INBA-863

#### How should this be manually tested?
run through the usages of the calendar. make sure it saves the values.

### CODE REVIEW

**Comment by laceyIrvin** [comment]
This will go beyond styling. The install works but there are some pieces we need to look at. I cant style this until its pulled out of the modal. We will need to sit with @jmfadeley  and talk about strategy for pulling these put of a modal and using a new page or overlay screen.

**Comment by morleegan** [comment]
@laceyIrvin - do you still want this PR approved and put in? Or should we wait?

**Comment by laceyIrvin** [comment]
Ill approve it but @jmfadeley should decide on whether to merge or not.

File: src/common/components/DateInput/index.js [hunk 1]
```diff
@@ -1,67 +1,40 @@
...
+import 'react-dates/initialize';
+import { DateRangePicker } from 'react-dates';
+import 'react-dates/lib/css/_datepicker.css';
 
 class DateInput extends Component {
     constructor(props) {
         super(props);
-
         this.state = {
-            expanded: false,
```
Comment (jmfadeley) [inline src/common/components/DateInput/index.js]: This is breaking the reuse of the `DateInput` at `src\views\TaskReview\components\Questions\Date.js` and at `src\views\TaskReview\components\Questions\Date.js`.

File: src/common/components/DateInput/index.js [hunk 2]
```diff
@@ -1,67 +1,40 @@
...
-                            onClick={this.expand}>
-                            {Time.renderCommon(value)}
-                            <IonIcon icon='ion-android-calendar'
-                                className='date-input__value-icon'/>
-                        </div>
-                }
+                <DateRangePicker
+                    startDate={this.state.startDate} // momentPropTypes.momentObj or null,
+                    startDateId="startDate" // PropTypes.string.isRequired,
+                    endDate={this.state.endDate} // momentPropTypes.momentObj or null,
```
Comment (jmfadeley) [inline src/common/components/DateInput/index.js]: In fact, the problem here is that questions and task details require a single calender view of the picker, not a double.

**Comment by jmfadeley** [comment]
@laceyIrvin @Arghast take a second gander at this PR now. I've pushed several changes because it was breaking the Task Details and the "date" type questions in they surveys. I also handled some styling for it that Lacey can fix later with ease but is presentable for now. To test, you need to:

1) Rerun `yarn` to remove the discarded `react-infinite-calendar`
2) Deploy. Create a project, and choose two dates using the new `react-dates` input while setting up a Stage for a Project.
3) Create a survey with at least one date type question. Assign it.
4) Go into the newly created task as the Administration and change the Task End Date (near the top). Ensure it works and sticks.
5) Login as the user, go and fill out the date for the survey. Ensure it saves properly.

**Comment by laceyIrvin** [comment]
@jmfadeley This looks good to me. I will make the styling ticket and add it to backlog for next sprint unless you have objections

**Pull request merged**